targets/BMS/main.cpp

```diff
@@ -75,9 +96,18 @@ int main() {
     // Queue that will store CANopen messages
     EVT::core::types::FixedQueue<CANOPEN_QUEUE_SIZE, IO::CANMessage> canOpenQueue;
 
+    // Intialize the system detect
+    BMS::DEV::SystemDetect systemDetect(BIKE_HEART_BEAT, CHARGER_HEART_BEAT,
+                                        DETECT_TIMEOUT);
+
+    // Create struct that will hold CAN interrupt parameters
+    struct CANInterruptParams canParams = {
+        .queue = &canOpenQueue,
+        .systemDetect = &systemDetect};
```

Comment:
NIT: Would look cleaner if there was a comma at the end of this line

Reply:
Good catch